src/Prometheus.MovieBuff.Presentation/Features/Movies/CreateMovie/v1/CreateMovieEndpoint.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace Prometheus.MovieBuff.Presentation.Features.Movies.CreateMovie.v1;
+
+internal static class CreateMovieEndpoint
+{
+    internal static IEndpointRouteBuilder MapCreateMovieEndpoint(this IEndpointRouteBuilder app)
+    {
+        app.MapPost("/movies", async (CreateMovieRequest request, CancellationToken cancellationToken = default) =>
+        {
+            var id = Guid.NewGuid();
+            var result = new { id = id };
+            // TODO => return Results.CreatedAtRoute("", )
+            return Results.Created($"/movies/{id}", result);
```

Comment:
lets not return this path as it does not yet exist
```suggestion
            return Results.Created();
```